fenix/app/src/main/res/values/strings.xml

```diff
@@ -393,21 +394,27 @@
     <string name="cookie_banner_handling_details_site_is_not_supported_request_support_button">Request support</string>
     <!-- Text for title indicating that cookie banner reduction is not supported for the current site, this is shown as part of the cookie banner details view. -->
-    <string name="cookie_banner_handling_details_site_is_not_supported_title">Cookie Banner Reduction</string>
+    <string name="cookie_banner_handling_details_site_is_not_supported_title" moz:RemovedIn="114" tools:ignore="UnusedResources">Cookie Banner Reduction</string>
+    <string name="cookie_banner_handling_details_site_is_not_supported_title_2">Request support for this site?</string>
     <!-- Label for the snackBar, after the user reports with success a website where cookie banner reducer did not work -->
-    <string name="cookie_banner_handling_report_site_snack_bar_text">Request to support site submitted.</string>
+    <string name="cookie_banner_handling_report_site_snack_bar_text" moz:RemovedIn="114" tools:ignore="UnusedResources">Request to support site submitted.</string>
+    <string name="cookie_banner_handling_report_site_snack_bar_text_2">Request sent</string>
     <!-- Text for indicating cookie banner handling is on this site, this is shown as part of the protections panel with the tracking protection toggle -->
     <string name="reduce_cookie_banner_on_for_site">On for this site</string>
     <!-- Text for indicating that a request for unsupported site was sent to Nimbus (it's a Mozilla library for experiments), this is shown as part of the protections panel with the tracking protection toggle -->
-    <string name="reduce_cookie_banner_unsupported_site_request_submitted">Request to support site submitted</string>
+    <string name="reduce_cookie_banner_unsupported_site_request_submitted" moz:RemovedIn="114" tools:ignore="UnusedResources">Request to support site submitted</string>
+    <string name="reduce_cookie_banner_unsupported_site_request_submitted_2">Support request sent</string>
     <!-- Text for indicating cookie banner handling is currently not supported for this site, this is shown as part of the protections panel with the tracking protection toggle -->
     <string name="reduce_cookie_banner_unsupported_site">Site currently not supported</string>
     <!-- Title text for a detail explanation indicating cookie banner handling is on this site, this is shown as part of the cookie banner panel in the toolbar. The first parameter is a shortened URL of the current site-->
-    <string name="reduce_cookie_banner_details_panel_title_on_for_site">Turn on Cookie Banner Reduction for %1$s?</string>
+    <string name="reduce_cookie_banner_details_panel_title_on_for_site" moz:RemovedIn="114" tools:ignore="UnusedResources">Turn on Cookie Banner Reduction for %1$s?</string>
+    <string name="reduce_cookie_banner_details_panel_title_on_for_site_2">Turn on Cookie banner reduction for %1$s?</string>
```

Comment:
Same here, no need to change string ID since only capitalization change (changing string ID means localizers will have to update the string again, and we let them chose capitalization/punctuation according to locale)

Reply:
Done